src/gui/search/searchwidget.h

```diff
@@ -82,6 +82,7 @@ private slots:
 
     void fillCatCombobox();
     void fillPluginComboBox();
+    void displayPage();
```

Comment:
It's true about the name. I couldn't imagine a suitable one.

Then maybe `void displayPage(bool hasInstalledPlugins)` ?